### ISSUE TITLE
Tizen 7.0 Items

### ISSUE THREAD

**Issue by zhoonit** [nnstreamer/nntrainer#1698]
1. Checkout delayed items from #1033 
2. related to #1516, add support for `ml_train_set_compile_properties()`, `ml_train_set_run_properties()` officially to overcome var_args limitation.

documentation
1. ini inference default behavior https://github.com/nnstreamer/nntrainer/pull/1711#discussion_r755199205

**Comment by myungjoo** [comment]
Tizen 7.0: (adding tizen-related item only)
- Tizen ML-Training-API in C# and Web

**Comment by kparichay** [comment]
- Learning rate scheduler for optimizer : #1776
